src/validation.cpp

```diff
@@ -3151,6 +3151,7 @@ bool ProcessNewBlockHeaders(const std::vector<CBlockHeader>& headers, CValidatio
 static bool AcceptBlock(const std::shared_ptr<const CBlock>& pblock, CValidationState& state, const CChainParams& chainparams, CBlockIndex** ppindex, bool fRequested, const CDiskBlockPos* dbp, bool* fNewBlock)
 {
     const CBlock& block = *pblock;
+    const uint256& blockHash = pblock->GetHash();
```

Comment:
Super nit: this may be slightly confusing as it may make it look like nothing new is computed, so I prefer `const uint256 hashBlock = ` here instead.